packages/contracts-bedrock/scripts/statediff.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+DEPLOY_MNEMONIC="test test test test test test test test test test test junk"
+MNEMONIC_PATH="m/44'/60'/0'/0/0"
+
+echo "> Deploying contracts to generate state diff (non-broadcast)"
+forge script -vvv scripts/Deploy.s.sol:Deploy --rpc-url "http://127.0.0.1:8545" --sig 'runWithStateDiff()' --mnemonic-passphrases "$DEPLOY_MNEMONIC" --mnemonic-derivation-paths "$MNEMONIC_PATH"
```

Comment:
Does this need the rpc url? Can't we omit that and run the whole thing in memory? Same with the mnemonic, if we omit that it can just work still with the default sender? The point of this script is to generate the state diff file that is committed into the repo right?